diagrams/sap/erp.py

```diff
@@ -0,0 +1,19 @@
+# This module is automatically generated by autogen.sh. DO NOT EDIT.
+
+from . import _SAP
+
+
+class _Erp(_SAP):
+    _type = "erp"
+    _icon_dir = "resources/sap/erp"
+
+
+class SAPS4HANACloud(_Erp):
+    _icon = "sap-s4hana-cloud.png"
+
+
+class SAPS4HANA(_Erp):
+    _icon = "sap-s4hana.png"
+
+
```

Comment:
In the resources folder there's a sub directory 'extensions' that is not listed here, actually Diagrams support only until 2 levels so the data inside extensions are skipped. 
I also noticed that there's already an extension directory with similar icons so if it's ok you can just delete the extensions directory.